src/command_modules/azure-cli-role/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -268,11 +276,13 @@ def test_graph_group_scenario(self):
             # check user1 memebership
             self.cmd('ad group member check -g {group} --member-id {user1_id}',
                      checks=self.check('value', True))
+
             # check user2 memebership
             self.cmd('ad group member check -g {group} --member-id {user2_id}',
-                     checks=self.check('value', True))            # list memebers
+                     checks=self.check('value', True))
+
             self.cmd('ad group member list -g {group}', checks=[
-                self.check("length([?displayName=='{user1}'])", 1),
```

Comment:
We can check if the older user1 name is not valid any more. Will something like below will do the check
self.check("length([?displayName=='{user1}'])", 0),